alert-engine/src/main/scala/io/phdata/pulse/alertengine/TriggeredAlert.scala

```diff
@@ -19,4 +19,7 @@ package io.phdata.pulse.alertengine
 import org.apache.solr.common.SolrDocument
 
 // Represents an alert that has been triggered. Contains information to notify based on the alert
-case class TriggeredAlert(rule: AlertRule, applicationName: String, documents: Seq[SolrDocument])
+case class TriggeredAlert(rule: AlertRule,
```

Comment:
Is there actually a newline here? Remove it if there is

Reply:
IntelliJ automatically changed the format to multi-line here, do you want to force keeping all them in one line?

Reply:
nope it's fine